azure-quantum/requirements-qiskit.txt

```diff
@@ -1,6 +1,5 @@
 qiskit-ionq>=0.3.3,<1.0
-# TODO(kuzminrobin, #494): See line below. Upon fix of #494 consider reverting fragment `<0.25.0` back to `<1.0`.
-qiskit-terra>=0.19.1,<0.25.0
 qiskit-qir>=0.4,<0.5
+qiskit>=0.45,<0.46
```

Comment:
limiting at `0.45.*` to make sure user doesn't pull breaking changes (potentially) when 0.46 (prep for Qiskit V1) gets released. We'll make another carefully tested release when Qiskit 0.46 comes out in Feb, 2024.